docs/src/modules/java/pages/timers.adoc

```diff
@@ -1,31 +1,11 @@
 = Timers
 :page-supergroup-java-scala: Language
 
-Timers allow for scheduling calls in the future. For example, to verify that some process have been completed or not.
+include::ROOT:partial$timers-intro.adoc[]
 
-Timers are persisted in the https://docs.kalix.io/reference/glossary.html#proxy[Kalix proxy] and are guaranteed to run at least once. 
+To demonstrate its functionality, let's consider an Ordering Service composed of a xref:value-entity.adoc[Value Entity] and an xref:actions.adoc[Action]. The Action will work as a xref:actions-as-controller.adoc[Controller] receiving calls and delegating to the Order Entity.
 
-When a timer is triggered, the scheduled call is executed. If successfully executed, the timer completes and is automatically removed.
-In case of a failure, the timer is rescheduled with a delay of 3 seconds. This process repeats until the call succeeds.
-
-You can schedule a timer for any service method in Kalix, but you can only create a timer from within an Action and by passing a `DeferredCall`.
-
-**Timer features**:
-
-* Timers are guaranteed to run at least once.
-* Timers can be scheduled to run at any time in the future.
-* Timers can be cancelled.
-* Timers are automatically removed once successfully completed.
-* Timers are re-scheduled in case of failures.
-
-**Timer limitations**: 
-
-* Timers can only be scheduled from within an Action.
-* Timers can only call other components (e.g., Actions and Entities), therefore external service calls must be wrapped by an Action in the deployed service.
-
-To demonstrate its functionality, let's consider an Ordering Service composed of a xref:value-entity.adoc[Value Entity] and an xref:actions.adoc[Action]. The Action will work as a xref:actions-as-controller.adoc[Controller] receiving calls and delegating to the Order Entity. 
-
-Users can place an order, but the order must be confirmed within a period of time. You can compare it with an Ordering Food application where the restaurant needs to confirm if it can accept the order. If no confirmation is sent within some pre-defined period of time, the order is automatically cancelled. 
+Users can place an order, but the order must be confirmed within a period of time. You can compare it with an Ordering Food application where the restaurant needs to confirm if it can accept the order. If no confirmation is sent within some pre-defined period of time, the order is automatically cancelled.
```

Comment:
I'm not sure how to rephrase it but `compare it with an Ordering Food application` seems not quite right. Especially the use of `compare`.  Maybe it would be better something like `think of it as part of an Ordering Food application`?

Reply:
agreed, changing